hr_holidays_imposed_days/models/hr_imposed_holidays.py

```diff
@@ -59,7 +59,11 @@ def validate(self):
                 leave._onchange_date_from()
                 created |= leave
             if rec.auto_confirm:
-                created.action_validate()
+                need_double_validation = created.filtered(
+                    lambda h: h.double_validation)
+                created.action_approve()
+                if need_double_validation:
+                    need_double_validation.action_validate()
```

Comment:
In order to get good coverage, you should add or change a test that would reach this line.
Otherwise, LGTM :-)